pkg/webui/console/containers/shortcut-panel/index.js

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2024 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { defineMessages } from 'react-intl'
+
+import Panel from '../../../components/panel'
+
+import ShortcutItem from './shortcut-item'
+
+const m = defineMessages({
+  shortcuts: 'Quick actions',
+  addApplication: 'Add a new Application',
+  addGateway: 'Add a new Gateway',
+  addOrganization: 'Add a new Organization',
+  addPersonalApiKey: 'Add a new personal API key',
+  registerDevice: 'Register a device',
```

Comment:
```suggestion
  shortcuts: 'Quick actions',
  addApplication: 'New Application',
  addGateway: 'New Gateway',
  addOrganization: 'New Organization',
  addPersonalApiKey: 'New personal API key',
  registerDevice: 'Register a device',
```
I've changed these to make them more concise. This is also updated in Figma.